### ISSUE TITLE
docs: when focused on TOC element, space scrolls page instead of going to link

### ISSUE THREAD

**Issue by jwunderl** [microsoft/pxt-microbit#2310]
**Describe the bug**

cannot use space to follow link when selected with tabbing; enter works

**To Reproduce**
1. Go to https://microbit.makecode.com/docs
2. tab to focus on table of contents link
3. press space

**Expected behavior**

should follow link

**Screenshots**

![2019-07-19 18 22 11](https://user-images.githubusercontent.com/5615930/61572451-b4456e00-aa52-11e9-9b04-f2f45f93c5fa.gif)

**Desktop (please complete the following information):**
 - OS: mac
 - Browser chrome

**Comment by pelikhan** [comment]
The TOC is mostly rewritten by the docs refactor.

________________________________
